<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvPrintCaptionPage</class>
 <widget class="QWidget" name="AdvPrintCaptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>845</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="Digikam::DItemsList" name="mPrintList" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Caption Type:</string>
              </property>
              <property name="buddy">
               <cstring>m_captionType</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_captionType"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="m_customCaptionGB">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,10,0,10">
                 <item row="3" column="1">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>print file name
print camera exposure time
print item comment
print ISO sensibility from camera
print date and time from camera</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="2">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>&quot;%r&quot; :
&quot;%a&quot; :
&quot;%l&quot; :
&quot;\n&quot; :</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>&quot;%f&quot; :
&quot;%t&quot; :
&quot;%c&quot; :
&quot;%i&quot; :
&quot;%d&quot; :</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="3">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>print camera sensor resolution
print camera aperture
print focal length from camera
add a new line</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="4">
                  <widget class="Digikam::DTextEdit" name="m_FreeCaptionFormat">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>400</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="placeholderText">
                    <string>Enter here your customized caption string...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" colspan="4">
                  <widget class="QLabel" name="m_free_label">
                   <property name="text">
                    <string>The fields listed below can be used with the customization:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Text Color:</string>
                  </property>
                  <property name="buddy">
                   <cstring>m_font_color</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Digikam::DColorSelector" name="m_font_color"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="textLabel1_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Text Size:</string>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                  <property name="buddy">
                   <cstring>m_font_size</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="m_font_size">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>8</number>
                  </property>
                  <property name="singleStep">
                   <number>2</number>
                  </property>
                  <property name="pageStep">
                   <number>2</number>
                  </property>
                  <property name="value">
                   <number>4</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Font Family:</string>
                </property>
                <property name="buddy">
                 <cstring>m_font_name</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFontComboBox" name="m_font_name"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Digikam::DColorSelector</class>
   <extends>QPushButton</extends>
   <header>dcolorselector.h</header>
  </customwidget>
  <customwidget>
   <class>Digikam::DItemsList</class>
   <extends>QWidget</extends>
   <header>ditemslist.h</header>
  </customwidget>
  <customwidget>
   <class>Digikam::DTextEdit</class>
   <extends>QTextEdit</extends>
   <header>dtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
